libs/portals/core/README.md

```diff
@@ -0,0 +1,7 @@
+# Portals Core
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
~Nice to have some description of what things the Core library contains and should contain.